### ISSUE TITLE
logrotation of logstash logs in Debian

### ISSUE THREAD

**Issue by adam-khan** [elastic/logstash#1574]
an aside: I tried to make an account on logstash.jira.com but Atlassian claims JIRA has no more licenses available.

logstash 1.4.2-1-2c0f5a1 from logstash's Debian repository
cron logrotate error:
error: skipping "/var/log/logstash/logstash.log" because parent directory has insecure permissions (It's world writable or writable by group which is not "root") Set "su" directive in config file to tell logrotate which user/group should be used for rotation.
I added
'su logstash adm' (I have logstash added to group adm on my system)
to /etc/logrotate.d/logstash
and the error disappears during execution of /etc/cron.daily/logrotate

**Comment by maq123** [comment]
It is very similiar issue like in https://github.com/elastic/logstash/issues/3490.
Basicaly it looks /etc/logrotate.d/logstash shipped within logstash deb packages ( I tested 1.5.2 and 1.5.3 ) has wrong file permissions. It has to be 644 to make it working.

My second suggestion is to cover more files into rotation. Not just *log one.
My corrected  /etc/logrotate.d/logstash looks like this  : 
```
 /var/log/logstash/*.log /var/log/logstash/*.err /var/log/logstash/*.stdout {
         daily
         rotate 7
         copytruncate
         compress
         delaycompress
         missingok
         notifempty
 }
```

**Comment by untergeek** [comment]
Closed by #4136
